bin/migration/tables/regions.py

```diff
@@ -0,0 +1,51 @@
+from .helpers import check_existing_record, audit_entry_creation, log_failed_imports
+import uuid
+
+class RegionManager:
+    def __init__(self):
+        self.failed_imports = set()
+
+    def migrate_data(self,destination_cursor):
+        batch_region_data = []
+        # Regions data -  https://cjscommonplatform-my.sharepoint.com/:x:/r/personal/lawrie_baber-scovell2_hmcts_net/_layouts/15/Doc.aspx?sourcedoc=%7B07C83A7F-EF01-4C78-9B02-AEDD443D15A1%7D&file=Courts%20PRE%20NRO.xlsx&wdOrigin=TEAMS-WEB.undefined_ns.rwc&action=default&mobileredirect=true
+        region_data = [
+            'London',
+            'West Midlands (England)',
+            'West Midlands (England)',
+            'East Midlands (England)',
+            'Yorkshire and The Humber',
+            'North East (England)',
+            'North West (England)',
+            'South East (England)',
+            'East of England',
+            'South West (England)',
+            'Wales'
+        ]
+
+        for region in region_data:
+            if not check_existing_record(destination_cursor,'regions', 'name', region):
```

Comment:
@mazzopardi2 why do we need regions table?

Reply:
I believe this is a new reporting requirement by the business. Best to be raised with Lawrie